extensions/functions_arithmetic.yaml

```diff
@@ -1214,20 +1214,6 @@ aggregate_functions:
         decomposable: MANY
         intermediate: fp64?
         return: fp64?
-      - args:
-          - name: x
-            value: timestamp
-        nullability: DECLARED_OUTPUT
-        decomposable: MANY
-        intermediate: timestamp?
-        return: timestamp?
-      - args:
-          - name: x
-            value: timestamp_tz
-        nullability: DECLARED_OUTPUT
-        decomposable: MANY
-        intermediate: timestamp_tz?
-        return: timestamp_tz?
```

Comment:
This is technically a breaking change. Plans may exist that use this version of the function, especially because it predates the version in `functions_datetime.yaml`.

API wise, it's probably better to remove these versions because the leaves all the time related `min` and `max functions in `functions_datetime.yaml`.

In terms of minimising breakage, it's probably better to remove the definition of these functions in `functions_datetime.yaml` as those have been around for less time.

I have a preference for removing them from `functions_datetime`, but it would be good to have other folks weigh in on this.